WooCommerce/Classes/ViewRelated/Orders/Refund/RefundSubmissionUseCase.swift

```diff
@@ -390,6 +398,21 @@ private extension RefundSubmissionUseCase {
         stores.dispatch(action)
         trackCreateRefundRequest()
     }
+
+    /// Retrieves the up-to-date refund data
+    /// - Parameters:
+    ///   - refund: the refund to retrieve details from.
+    ///   - onCompletion: called when the submission completes.
+    func getLatestRefundData(refund: Refund, onCompletion: @escaping (Result<Void, Error>) -> Void) {
```

Comment:
I see that this function is just dispatching the `retrieveRefund` action, can we call that one directly without the need to create another function?
Otherwise, if you just want that function to encapsulate the action call it is fine :) In that case, I would:
- Make it private
- Remove the completion parameter as it seems like we do not need it :)